prowler/lib/outputs/html/html.py

```diff
@@ -0,0 +1,567 @@
+import html
+import importlib
+import sys
+from os import path
+
+from prowler.config.config import (
+    html_file_suffix,
+    html_logo_url,
+    prowler_version,
+    square_logo_img,
+    timestamp,
+)
+from prowler.lib.logger import logger
+from prowler.lib.outputs.compliance.compliance import get_check_compliance
+from prowler.lib.outputs.utils import parse_html_string, unroll_dict
+from prowler.lib.utils.utils import open_file
+
+
+def add_html_header(file_descriptor, provider):
+    try:
+        parameters = sys.argv[1:]
+        for index, parameter in enumerate(parameters):
+            if (
+                parameter == "--kubeconfig-file"
+                and "/.kube/config" in parameters[index + 1]
+            ):
+                parameters[index + 1] = "~/.kube/config"
```

Comment:
Why this? Please leave this function jus to fill the header, if you need to do some other changes pass do them in another place.

Reply:
Done